app/controllers/patients_controller.rb

```diff
@@ -19,7 +19,7 @@ def show
     @patient = current_user.get_patient(params.permit(:id)[:id])
 
     # If we failed to find a subject given the id, redirect to index
-    redirect_to(root_url) && return if @patient.nil?
+    redirect_to(action: 'unknown_patient') && return if @patient.nil?
```

Comment:
I don't love this file name.  maybe `monitoree_unavailable` or `patient_unavailable` to match the title on the page?